src/js_parser.zig

```diff
@@ -17355,9 +17356,151 @@ fn NewParser_(
         ) []Stmt {
             switch (stmtorexpr) {
                 .stmt => |stmt| {
```

Comment:
are decorators with class expressions supported? this would only run for class statements

Reply:
They aren't supported, but in the future they might be.
https://github.com/microsoft/TypeScript/issues/7342